docs/pages/brownfield/integration-with-existing-apps.mdx

```diff
@@ -0,0 +1,619 @@
+
+```rb Podfile
+require File.join(File.dirname(`node --print "require.resolve('expo/package.json')"`), "scripts/autolinking")
+require File.join(File.dirname(`node --print "require.resolve('react-native/package.json')"`), "scripts/react_native_pods")
+
+require 'json'
+
+platform :ios, '15.1'
+install! 'cocoapods',
+  :deterministic_uuids => false
+
+prepare_react_native_project!
+
+target 'HelloWorld' do
+  use_expo_modules!
+
+  config_command = [
+    'npx',
+    'expo-modules-autolinking',
+    'react-native-config',
+    '--json',
+    '--platform',
+    'ios'
+  ]
+  config = use_native_modules!(config_command)
+
+  use_frameworks! :linkage => ENV['USE_FRAMEWORKS'].to_sym if ENV['USE_FRAMEWORKS']
+
+  use_react_native!(
+    :path => config[:reactNativePath],
+    :hermes_enabled => true,
+    # An absolute path to your application root.
+    :app_path => "#{Pod::Config.instance.installation_root}/..",
+    :privacy_file_aggregation_enabled => true,
+  )
+
+  post_install do |installer|
+    react_native_post_install(
+      installer,
+      config[:reactNativePath],
+      :mac_catalyst_enabled => false,
+    )
+  end
+end
+```
+
+If your project already has a **Podfile**, you'll need to manually merge the React Native dependencies into your existing **Podfile**.
+
+If you're using a custom folder structure, you need to explicitly set your project root in **Podfile** for autolinking to work. Modify the following lines in your **Podfile**:
+
+```diff Podfile
++project_root = File.join(__dir__, 'exp')
++ENV['PROJECT_ROOT'] = project_root
+
+target 'HelloWorld' do
+-  use_expo_modules!
++  use_expo_modules!({ projectRoot: project_root })
+
+config_command = [
+    'npx',
+    'expo-modules-autolinking',
+    'react-native-config',
+    '--json',
+    '--platform',
+-    'ios'
++    'ios',
++    '--project-root',
++    project_root,
++    '--source-dir',
++    "#{__dir__}"
+  ]
+
+  use_react_native!(
+    :path => config[:reactNativePath],
+    :hermes_enabled => true,
+    # An absolute path to your application root.
+-    :app_path => "#{Pod::Config.instance.installation_root}/..",
++    :app_path => "#{project_root}",
+    :privacy_file_aggregation_enabled => true,
+  )
+```
+
+Now, run the following command:
+
+<Terminal cmd={['$ pod install']} />
+
+Running the `pod` command will integrate the React Native code into your app, allowing your iOS files to import the React Native headers.
+
+### Configuring your Xcode project
+
+<Step label="1">
+  Open your project on Xcode and from the project navigator, select your project and then select
+  your app target under `TARGETS`. In **Build Settings**, using the search bar, search for
+  `ENABLE_USER_SCRIPT_SANDBOXING`. If it is not already, set its value to `No`. This is needed to
+  properly switch between the Debug and Release versions of the [Hermes
+  engine](https://github.com/facebook/hermes/blob/main/README.md) that is shipped with React Native.
+</Step>
```

Comment:
maybe add a screenshot would help to explain the this better. xcode setting is sometimes too complicated to find where to set this up